test/testrunner.py

```diff
@@ -316,7 +316,9 @@ def main():
         sm_utils.generate_empty_report(report, test_type, "huggingface")
         return
 
-    if specific_test_type in ("sanity", "ecs", "ec2", "eks", "canary", "bai", "quick_checks"):
+    if specific_test_type in (
+            "sanity", "ecs", "ec2", "eks", "canary", "bai", "quick_checks", "release_candidate_integration"
```

Comment:
could you also add _"release_candidate_integration"_ to if block at line 313?

Reply:
Added, thanks for pointing that out

Reply:
Sorry, didn't push "Add comment"
I don't see any handlers for "release_candidate_integration" down there. Is it supposed to be added later on?

Reply:
Not sure I understand - are you referring to the finally block around line 422? There shouldn't need to be any specific RC integration handlers here afaik

Reply:
Oh, maybe you mean why do we not have any "if" conditions for these tests? This is because they run with the default behavior 

```
pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto"]
```

so there is no need to add

Reply:
From 334 till 407 we're creating a pytest command based on test type and other parameters. Since you don't have a special "if" block for "release_candidate_integration", you'll end up with a command created at [402](https://github.com/aws/deep-learning-containers/pull/1178/files#diff-7cbc819cfee09f8be65c72606589778a1d44839986c87dfb0930a40ab75820efR402). 
So if you need any special/custom parameter to be added to the command you'll need an additional handler.

Reply:
Nothing else is needed, default path is fine as mentioned above. Thanks for checking this